### ISSUE TITLE
Readme what's what

### ISSUE THREAD

**Issue by svbeon** [freenode/freenodejobs#21]
Well if I'm allowed to be a bit picky, as we're basooning this repo to the greater public, perhaps describing in the readme what goes in what folder, or basically a brief explanation of how this is structured and why some folders etc. Are there.

The logs link might also not be preferable to share publicly (even tough the ACL blocks the wrong people from actually seeing the logs)

And maybe, just maybe, we should make the repo follow the [code style guidelines](https://github.com/freenode/web-7.0/blob/master/CONTRIBUTING.md) we use for web-7.0, where applicable?

Just some open questions, feel free to shoot them down :)

**Comment by lamby** [comment]
Very happy to write documentation. However, I've naturally got a bit of the "curse of knowledge" in that I'm not sure exactly what documentation — and what level of documentation — would be most effective. Can you create a quick list that I can go through and check off? That would move this issue from something fairly vague to an actionable item :)

**Comment by svbeon** [comment]
well, this might sound somewhat abstract still, but to me a basic directory tree with a sentence or two per directory explaining what's in there. Like for the templates folder, what template format is it using (django's?). the media/css/freenodejobs.d, i guess those files are combined and processed, is it less, sass, cssnext? and same for the js folder? the debian and config folders, is that only for deploying with ansible or is there config values you need to worry about when setting it up locally? same for the debian folder

basically someting like that is what i'd be looking for

**Comment by svbeon** [comment]
🎉
